### DISCUSSION TITLE
ERROR 2003 (HY000): Can't connect to MySQL server on 'mysql.devops72bat.online:3306'

### DISCUSSION THREAD

**Discussion by prabalark** [learndevopsonline/learndevopsonline#1903] [Q&A]
hi team ,
in variable ,password given as input for shipping.sh , getting below error at mysql install
![image](https://github.com/learndevopsonline/learndevopsonline/assets/130270906/cf2f38a4-c108-4aaf-a2fa-f3e28db38368)

**Comment by chaithanya1812** [comment]
Hi @prabalark,
### IMP:- Follow the guidelines how to post an error? 
### Issue:- Your mysql server is not running, check your mysql-server & route53-records.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/f7f474cc-8507-48ea-8afb-cfde2300ae19)

**Reply by prabalark** [reply]
ok chaitanya
